packages/server-wallet/src/wallet/__test__/integration/close-channel.test.ts

```diff
@@ -1,38 +1,50 @@
+import Knex from 'knex';
+
 import {Channel} from '../../../models/channel';
 import {Wallet} from '../..';
 import {seedAlicesSigningWallet} from '../../../db/seeds/1_signing_wallet_seeds';
 import {truncate} from '../../../db-admin/db-admin-connection';
-import knex from '../../../db/connection';
 import {stateWithHashSignedBy} from '../fixtures/states';
 import {alice, bob} from '../fixtures/signing-wallets';
 import {channel} from '../../../models/__test__/fixtures/channel';
+import {defaultConfig, extractDBConfigFromServerWalletConfig} from '../../../config';
+
+const knex: Knex = Knex(extractDBConfigFromServerWalletConfig(defaultConfig));
```

Comment:
Could this use `testKnex`?

Reply:
This spawns a `new Wallet`, so it could use that wallet's `knex` exclusively.

Reply:
b7d7eb7